README.md

```diff
@@ -1,6 +1,6 @@
 # ➤ bchydro-outages
 
-This is an *unofficial* repository that provides a Git history of outages at [BC Hydro](https://bchydro.com), a Canadian utility.
+This is an *unofficial* repository that provides a Git history of outages at [BC Hydro](https://bchydro.com), a Canadian utility company.
```

Comment:
I think we can leave the original language.

Reply:
A utility would be "gas" or "electricity" as a purchasable thing, whereas a utility company would be "BC Hydro" or "FortisBC" no?